static/css/theme-activestate.css

```diff
@@ -12,6 +12,7 @@
     --MAIN-LINK-color:#2D52BD; /* Color of links */
     --MAIN-LINK-HOVER-color:#2D52BD; /* Color of hovered links */
     --MAIN-ANCHOR-color: #2D52BD; /* color of anchors on titles */
+    --MAIN-BG-color: #DAEAFF; /*color of code snippet backgrounds */
 
     --MENU-HEADER-BG-color:#FF2F4C; /* Background color of menu header */
     --MENU-HEADER-BORDER-color:#FF2F4C; /*Color of menu header border */ 
```

Comment:
Can you standardize the capitalization & spacing of the comments for this block? One other thing to make it more readable is to tab in all of the comments so they start at the same spot like...

```
--MAIN-LINK-color:#2D52BD;          /* Color of links */
--MAIN-LINK-HOVER-color:#2D52BD;    /* Color of hovered links */
--MAIN-ANCHOR-color: #2D52BD;       /* Color of anchors on titles */
```

Optional but I think it looks nicer.